main.tf

```diff
@@ -124,6 +124,7 @@ module "dcos-infrastructure" {
   public_agents_os                           = "${var.public_agents_os}"
   public_agents_root_volume_size             = "${var.public_agents_root_volume_size}"
   public_agents_root_volume_type             = "${var.public_agents_root_volume_type}"
+  public_agents_ips                          = ["${var.public_agents_ips}"]
```

Comment:
Hmmm... Probably a better name could be used here. Was using this since it existed already in the SG module already. 

Perhaps like `public_agents_access_ips` to keep in tune with public agents vars and in tune with admin_ips?!?!

Reply:
LGTM